exonum-java-binding/core/pom.xml

```diff
@@ -100,7 +100,13 @@
 
     <dependency>
       <groupId>com.ryanharter.auto.value</groupId>
-      <artifactId>auto-value-gson</artifactId>
+      <artifactId>auto-value-gson-extension</artifactId>
+      <scope>provided</scope>
```

Comment:
as far as I remember `scope` inherits from the parent dependency management. i.e. is not required to be defined explicitly

Reply:
👍 
I'd probably keep it for higher clarity.